coverage_test.py

```diff
@@ -961,7 +961,29 @@ def test_options(self):
         exchange3 = (req, expected)
         self.current_mid += 1
 
-        self._test_with_client([exchange1, exchange2, exchange3])
+        req = Request()
+        req.code = defines.Codes.POST.number
+        req.uri_path = path
+        req.type = defines.Types["CON"]
+        req._mid = self.current_mid
+        req.destination = self.server_address
+        option = Option()
+        option.number = defines.OptionRegistry.TIME.number
+        req.add_option(option)
+        req.payload = "test"
+
+        expected = Response()
```

Comment:
Perhaps add a comment here similar to what you have in the PR description that we're not expecting a Time option in the response since this does not run against a Hono server... so the purpose of the test is more to verify that adding the option to the request does not result in any unexpected weirdness.